packages/contracts-bedrock/scripts/checks/check-interfaces.sh

```diff
@@ -208,6 +208,29 @@ for interface_file in $JSON_FILES; do
     normalized_interface_abi=$(normalize_abi "$interface_abi")
     normalized_contract_abi=$(normalize_abi "$contract_abi")
 
+    # Check if the contract ABI has no constructor but the interface is missing __constructor__
+    contract_has_constructor=$(echo "$normalized_contract_abi" | jq 'any(.[]; .type == "constructor")')
+    interface_has_pseudo_constructor=$(echo "$normalized_interface_abi" | jq 'any(.[]; .type == "constructor")')
+
+    # If any contract has no constructor and its corresponding interface also does not have one, flag it as a detected issue
+    if [ "$contract_has_constructor" = false ] && [ "$interface_has_pseudo_constructor" = false ]; then
```

Comment:
Maybe we can explicitly check for a `$contract_has_constructor = false && $interface_has_empty_pseudo_constructor`